.github/workflows/release.yaml

```diff
@@ -123,6 +127,55 @@ jobs:
       - name :  docker images publish
         run: |
           make docker-publish-cli
+      
+      - name: Make Release
+        env:
+          VERSION: ${{ steps.version.outputs.version }}
+        run: |
+          rm -rf release
+          make release
+          make release-notes > release/release-notes.out
+          cat release/release-notes.out
+          
+      - name: Run GoReleaser
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: release --rm-dist --debug --release-notes=release/release-notes.out
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  
+  create-release:
+    runs-on: ubuntu-latest
+    needs:
```

Comment:
The create-release workflow only needs to depend on the other release workflows if you want those failures to stop the release page and what not from getting created.  There is no direct need for these release workflows to come before releasing the release page and and publishing tar.gz binaries.